sftpgo.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: sftpgo
+  version: 2.6.2
+  epoch: 0
+  description: Full-featured and highly configurable SFTP, HTTP/S, FTP/S and WebDAV server - S3, Google Cloud Storage, Azure Blob
+  copyright:
+  - license: AGPL-3.0-only
+  dependencies:
+    provides:
+    - sftpgo=${{package.full-version}}
+    runtime:
+    - wolfi-baselayout
+environment:
+  contents:
+    packages:
+    - ca-certificates-bundle
+    - go
+    - gcc
+    - git
+    - busybox
+    - wolfi-baselayout
+    - curl
+
+pipeline:
+- uses: git-checkout
+  with:
+    repository: https://github.com/drakkan/sftpgo
+    tag: v${{package.version}}
+    expected-commit: 636a1c2c385a16966842da4acad5acd36163ffb4
+
+- runs: |
+    # Modify the default configuration file
+    sed -i 's|"users_base_dir": "",|"users_base_dir": "/srv/sftpgo/data",|' sftpgo.json && \
+    sed -i 's|"backups"|"/srv/sftpgo/backups"|' sftpgo.json && \
+    sed -i 's|"sqlite"|"bolt"|' sftpgo.json
+
+- runs: |
+    export GOFLAGS="-mod=readonly"
+    go mod download
+    set -xe && \
+    export COMMIT_SHA=${COMMIT_SHA:-$(git describe --always --abbrev=8 --dirty)} && \
+    go build -trimpath -ldflags "-s -w -X github.com/drakkan/sftpgo/v2/internal/version.commit=${COMMIT_SHA} -X github.com/drakkan/sftpgo/v2/internal/version.date=`date -u +%FT%TZ`" -v -o sftpgo
+    #Download official plugins
+    mkdir -p /usr/local/bin && ./docker/scripts/download-plugins.sh
+
+- runs: |
+    mkdir -p ${{targets.destdir}}/etc/sftpgo ${{targets.destdir}}/var/lib/sftpgo ${{targets.destdir}}/usr/share/sftpgo ${{targets.destdir}}/srv/sftpgo/data ${{targets.destdir}}/srv/sftpgo/backups ${{targets.destdir}}/usr/local/bin
+    cp sftpgo.json ${{targets.destdir}}/etc/sftpgo/sftpgo.json
+    cp -r templates ${{targets.destdir}}/usr/share/sftpgo/templates
+    cp -r static ${{targets.destdir}}/usr/share/sftpgo/static
+    cp -r openapi ${{targets.destdir}}/usr/share/sftpgo/openapi
+    cp sftpgo ${{targets.destdir}}/usr/local/bin/
+    cp /usr/local/bin/sftpgo-plugin-* ${{targets.destdir}}/usr/local/bin/
```

Comment:
move stuff under `usr/local` to `sftpgo-compat` package maybe?

```
2024/08/08 00:42:40 WARN linter "usrlocal" failed on package "sftpgo": /usr/local path found in non-compat package: usr/local/bin/sftpgo; suggest: This package should be a -compat package
```

Reply:
I already tried to switch to /usr/bin to fix that linter, still not work.

Reply:
Move them to subpkg sftpgo-compat